### PR TITLE
Honkerprise; A new frontier

### PR DESCRIPTION
**Pull request by GoatMonarch**
After a month of mapping (and copy pasting objects from the Aetherwhisp), constant crying myself to sleep over byond's pathing, and shitposting in our discord, the Honkerprise is actually functional for people other than me meaning it's ready for an overview!

The Honkerprise is a ship mostly made out of bananium to introduce more people to the !!fun!! that is having squeaky clown shoes, and the constant sound of bike horns.
In addition I have worked to implement one item(s) for each department job of which Altangy has overview and approved of. I also have a txt lining out all of the items if requested. 

A few examples are: Atmospehics having two meme gass tanks. HoS having a cluster teargass grenade. Robotics having one stack of bananium so they won't tear up the ship for materials. Clown having 300 creampies. Security being partly a satelite so breakins and break outs will be more difficult.

As such the honkerprise would be a great admin event only ship for it brings a normal ordinary and functional ship, accompanied with toys for each job. 

There are also some references hidden around the ship, two of them being from the FTL winter logo contest [one of the three winners did not return about winning the contest] along with general input and ideas from players active in the discord.

why: This originaly started off from me losing a bet with Crossed, leading to me creating a complete ship, which later continued onwards for me to get mapping experience, and to create a fun map where hopefully players will be happy to find out what their favorite job's item may be.

Also apparantly people want to see what the map looks like through the PR, so there you go.
honkerprise 1 0 no areas
honkerprise 1 0

### CODE REVIEW

**Comment by GoatMonarch** [comment]
Allright people that's going to test the map, please fill this out so I can just more easily add it to a list of how to fix your problems.

Room name:
Object name:
Describe the issue:
Incase of no object, describe the area the issue stands from:

**Comment by SgtxRootbeer** [comment]
Not enough lizard
:Thumbs down:

**Comment by GoatMonarch** [comment]
There's also an issue related to the AI slippers not following the correct access, so they are to be removed. If anybody locates any feel free to just name the rooms they are in.

**Comment by NIXC** [comment]
Room name: ~Cargo shuttle
Object name: reinforced windows
Describes-The-Issue: the left top section does not have an area assigned, and thus has funky lighting.
image

**Comment by GoatMonarch** [comment]
> Room name: ~Cargo shuttle
> Object name: reinforced windows
> Describes-The-Issue: the left top section does not have an area assigned, and thus has funky lighting.
> image

Solved the issue thanks for finding that!



> Room name: Dormitories/Cryogenics
> Object name: Air alarm
> Describe the issue: This room is missing an air alarm. Some areas in maintenance are also missing air alarms.

Kinda missed the later part of that issue due to someone making two issues at the same time HONK!

**Comment by Karmic-Skink** [comment]
Lets test merge it and take it for a spin next weekend. If there are any issues, I'm sure people will find them with it a real crew on it.

**Comment by GoatMonarch** [comment]
> Lets test merge it and take it for a spin next weekend. If there are any issues, I'm sure people will find them with it a real crew on it.

Fairly sure alot of players would love to do that, though we'll need Floyd's approval and hopefully having some mapper or two just chip on in and officialy say they think this is test merge ready. What is clear is that you could play a round on it with no problems atleast!

**Comment by NIXC** [comment]
I'ts definitely test merge ready. Considering its more of a special occasion map being fully merged probably wouldn't hurt, its not like we can't change the map if we find something actually wrong.

**Comment by GoatMonarch** [comment]
> I'ts definitely test merge ready. Considering its more of a special occasion map being fully merged probably wouldn't hurt, its not like we can't change the map if we find something actually wrong.

Thanks for chipping in! Allthough as for the merge it'd probably be wise to wait untill crossed has returned from his vacation since he had somewhat of an idea of chaging config files to ensure he can just flip a switch whenever even admins are able to switch to the map.
_Though not like we'd have anything but the honk ship over the weekend it's merged_

**Comment by crossedfall** [comment]
I wish I was on vacation. That would have made life a lot easier. I'll mark it for a test merge and we can throw it up for a round or two on Friday to identify lingering issues.

**Comment by GoatMonarch** [comment]
> I wish I was on vacation. That would have made life a lot easier. I'll mark it for a test merge and we can throw it up for a round or two on Friday to identify lingering issues.
> 
> Edit: Since this is being test merged. I'm shutting down the map-test server since we don't need to mirror things over anymore.

Aha, just a-help for a "vacation token" some time and you'll probably get it HONK!
Though glad to hear the test merge has come thru, I'll look forward to a week of solving mapping issues and explaining why two chairs are "floating" in "space". Also thanks for the test server it had been great help for quick player input!

File: _maps/Honkerprise.dm [hunk 1]
```diff
@@ -0,0 +1 @@
+#define FORCE_MAP "_maps/Honkerprise.json"
```
Comment (Kmc2000) [inline _maps/Honkerprise.dm]: If this is merged, you'll only be able to play this map

**Comment by GoatMonarch** [comment]
@ondrej008 Not entirely sure, though I have stopped geting input related to map issues which normaly would mean nobody plays on it, but as it's been used on several occasions since the first weekend I suspect it's safe to say that the map is onpar with the quality of the Aetherwhisp.

So I'll just put it down to inactivity on the github as for it not being fully in, not as though it matters, map's still aviable for players whenever an admin decides so.

**Comment by Kmc2000** [comment]
Literally scarab v2 well done 💯

**Comment by GoatMonarch** [comment]
But the scarab once test merged was a broken mess and still is to some extent reee

**Comment by crossedfall** [comment]
We're EoL and you deserve this.

**Pull request merged**